docker-java-api/pom.xml

```diff
@@ -4,7 +4,7 @@
 	<parent>
 		<groupId>com.github.docker-java</groupId>
 		<artifactId>docker-java-parent</artifactId>
-		<version>3.2.2-SNAPSHOT</version>
+		<version>3.2.6-SNAPSHOT</version>
```

Comment:
could you please revert the version change?

Reply:
I can, but why? It actually makes it harder for me to do local build of this and test it against STS (which is where we use it). The wrong version will mess up my local setup that consumes the dependencies.

I would suggest that its better to strive for an accurate version number in there. I know you override the version at build-time so it doesn't matter what you put in there (so again, makes me wonder why you'd care if I 'fix it' to be accurate. Why would you insist on keeping an old version in there? There seems to be no benefit to that whatsoever).

Reply:
> The wrong version will mess up my local setup that consumes the dependencies.
> insist on keeping an old version in there

soon, there will be no "current" version at all, just `0-SNAPSHOT`. So I am insisting on not touching the version at all, because this is not feature PR's responsibility.

> makes me wonder why you'd care if I 'fix it' to be accurate

Even if you're willing to "fix it", that would be a separate PR that changes it to `0-SNAPSHOT`, according to the plan.

Reply:
Okay. I will do as you wish of course. But please allow me to address this comment as the very last one. 

I.e. once everything else about this PR is acceptable, then I will address this comment. If I were to do it now, it just adds a bunch friction around the local process I'm currently needing to follow to validate that the PR actually works for us.

Reply:
Change reverted